tests/unit/deploy/test_planner.py

```diff
@@ -672,6 +798,51 @@ class Foo(models.ManagedModel):
         with pytest.raises(ValueError):
             self.remote_state.resource_exists(foo)
 
+    @pytest.mark.parametrize(
+        'resource_topic,deployed_topic,resource_exists,expected_result', [
```

Comment:
I wonder if it makes sense to reword `resource_exists` to something to indicate it is the return value from `verify_sns_subscription_current`? I got hung up a little bit on this as just reading the parameter names it seemed redundant to have a `resource_exists` and `expected_result` parameter for something that was testing if the subscription exists. It seemed like only one would have been needed.